fastlane/Fastfile

```diff
@@ -495,3 +501,29 @@ lane :download_strings_files_from_glotpress do
     download_dir: File.join(PROJECT_FOLDER, 'Simplenote')
   )
 end
+
+# Downloads all the required certificates and profiles for the production build.
+# Optionally, it can create any new necessary certificates or profiles.
+#
+# @option [Boolean] readonly (default: true) Whether to only fetch existing certificates and profiles, without generating new ones.
+lane :app_store_code_signing do |options|
+  # Fail early if secrets not available via `get_required_env`.
+  # Otherwise, Fastlane will prompt to type them.
+  access_key = get_required_env('MATCH_S3_ACCESS_KEY')
+  secret_access_key = get_required_env('MATCH_S3_SECRET_ACCESS_KEY')
+
+  match(
+    storage_mode: 's3',
+    s3_bucket: 'a8c-fastlane-match',
+    s3_access_key: access_key,
+    s3_secret_access_key: secret_access_key,
```

Comment:
I got this while testing

![image](https://github.com/Automattic/simplenote-macos/assets/1218433/ff25d813-a1e2-49e9-84da-d9d65fad1ef5)

Configuring the region as well fixed it.

<img width="1197" alt="image" src="https://github.com/Automattic/simplenote-macos/assets/1218433/04c78e56-1dbb-4639-a937-0b30f4e24317">

---

```suggestion
    s3_region: 'us-east-2',
```

Reply:
Fixed in https://github.com/Automattic/simplenote-macos/commit/0c40b68a3d58ba47f531d2ce2439b073731d1a95